src/lib/io/include/sol-i2c.h

```diff
@@ -326,21 +336,12 @@ struct sol_i2c_pending *sol_i2c_write_register(struct sol_i2c *i2c, uint8_t reg,
  * @return pending An I2C pending operation handle on success,
  * otherwise @c NULL. It's only valid before @a read_reg_multiple_cb
  * is called. It may be used before that to cancel the read operation.
+ * If @c NULL is returned, the errno variable will be set with the correct error value. The error value is always negative.
+ * In case that the AIO device is in use, the errno variale is set to -EBUSY.
  */
 struct sol_i2c_pending *sol_i2c_read_register_multiple(struct sol_i2c *i2c, uint8_t reg, uint8_t *values, size_t count, uint8_t times, void (*read_reg_multiple_cb)(void *cb_data, struct sol_i2c *i2c, uint8_t reg, uint8_t *data, ssize_t status), const void *cb_data);
 
 /**
- * @brief Return true if I2C handle's bus is busy, processing another
- * operation. This function should be called before issuing any other
- * I2C function.
- *
- * @param i2c The I2C bus handle
- *
- * @return true is busy or false if idle
- */
-bool sol_i2c_busy(struct sol_i2c *i2c);
-
```

Comment:
We need to remove this from the low level bindings(i.e i2c.js ) also and adjust the JS code a bit. I can do that after this is landed.